src/openrct2/Game.cpp

```diff
@@ -173,7 +173,7 @@ void RCT2StringToUTF8Self(char* buffer, size_t length)
     if (length > 0)
     {
         auto temp = RCT2StringToUTF8(buffer, RCT2LanguageId::EnglishUK);
-        SafeStrCpy(buffer, temp.data(), length);
+        String::SafeStrCpy(buffer, temp.data(), length);
```

Comment:
Should we maybe rename it? I don't like reading this, since it also touches all users would be not the worst idea to fix the style as in camelCase